scripts/ableplayer-base.js

```diff
@@ -68,10 +68,14 @@ exports.initAllAblePlayers = function () {
 	// Construct an AblePlayer object
 	// Parameters are:
 	// media - jQuery selector or element identifying the media.
-	window.AblePlayer = function (media) {
+	window.AblePlayer = function (media, onLoaded) {
```

Comment:
Maybe better to have `onLoaded` inside of a `options` object parameter? So we can add more options later if we want.

Reply:
Mmm there are 2 places that uses `new AblePlayer(...)` in this repo... does Javascript allow default input arg? Then we can default `options` to empty..?  Coz at this rate it'll break `video13.html` and `exports.initAllAblePlayers` in `ableplayer-base.js`

Reply:
we can do something like
```
if (typeof options === 'undefined') {
  this.onLoaded = options.onLoaded;
} else {
  this.onLoaded = null;
}
```

And we should check if `thisObj.onLoaded` exists before calling it.

Reply:
really good point kelvin thanks :)